packages/owl-bot/src/owl-bot.ts

```diff
@@ -255,4 +161,132 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       Number(context.payload.installation.id)
     );
   });
+}
+
+export async function handlePullRequestLabeled(
```

Comment:
This code was pulled out from an anonymous closure, and exported so that it can be used by the CLI. No code changes.